### PR TITLE
fix: Speakers: Rename column rank-> order

### PR DESCRIPTION
**Pull request by MeghalBisht**
Related #7576 

#### Short description of what this resolves:
self explanatory

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

File: app/api/schema/speakers.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ class Meta:
     country = fields.Str(allow_none=True)
     city = fields.Str(allow_none=True)
     gender = fields.Str(allow_none=True)
-    rank = fields.Integer(allow_none=True, default=0)
+    order = fields.Integer(allow_none=True, default=0)
```
Comment (iamareebjamal) [inline app/api/schema/speakers.py:56]:
```suggestion
    position = fields.Integer(allow_none=True, default=0)
```

File: app/models/speaker.py [hunk 2]
```diff
@@ -33,7 +33,7 @@ class Speaker(SoftDeletionModel):
     country = db.Column(db.String)
     city = db.Column(db.String)
     gender = db.Column(db.String)
-    rank = db.Column(db.Integer, default=0, nullable=False)
+    order = db.Column(db.Integer, default=0, nullable=False)
```
Comment (iamareebjamal) [inline app/models/speaker.py:36]:
```suggestion
    position = db.Column(db.Integer, default=0, nullable=False)
```

Comment (MeghalBisht) [reply app/models/speaker.py:36]: position is already a column

File: migrations/versions/rev-2021-01-07-05:19:49-3b29ea38f0cb_.py [hunk 3]
```diff
@@ -18,11 +18,11 @@
 
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
-    op.add_column('speaker', sa.Column('rank', sa.Integer(), nullable=False, server_default='0'))
+    op.add_column('speaker', sa.Column('order', sa.Integer(), nullable=False, server_default='0'))
```
Comment (iamareebjamal) [inline migrations/versions/rev-2021-01-07-05:19:49-3b29ea38f0cb_.py:21]: Why adding new column, you have to rename the rank column to order

Comment (MeghalBisht) [reply migrations/versions/rev-2021-01-07-05:19:49-3b29ea38f0cb_.py:21]: So i need to run the migrations again?

Comment (iamareebjamal) [reply migrations/versions/rev-2021-01-07-05:19:49-3b29ea38f0cb_.py:21]: Changing migrations don't work. Once a migration is committed, it is run and is permanently immutable, you have to create a new migration

Comment (MeghalBisht) [reply migrations/versions/rev-2021-01-07-05:19:49-3b29ea38f0cb_.py:21]: Okay and position is already a column, shall I change back to order?

Comment (iamareebjamal) [reply migrations/versions/rev-2021-01-07-05:19:49-3b29ea38f0cb_.py:21]: Ahh, ok. Yes

**Pull request merged**